include/alpaka/dev/cpu/SysInfo.hpp

```diff
@@ -123,6 +123,26 @@ namespace alpaka
                 return std::string("unknown");
 #endif
             }
+
+            //! \return Pagesize in bytes used by the system.
+            inline size_t getPageSize()
+            {
+#if BOOST_OS_WINDOWS || BOOST_OS_CYGWIN
+                SYSTEM_INFO si;
+                GetSystemInfo(&si);
+                return si.dwPageSize;
+#elif BOOST_OS_UNIX || BOOST_OS_MACOS
+#    if defined(_SC_PAGESIZE)
+                return static_cast<std::size_t>(sysconf(_SC_PAGESIZE));
+#    else
+                return = static_cast<size_t>(getpagesize());
```

Comment:
Are there still systems around which don't use `_SC_PAGESIZE`? `getpagesize()` was deprecated [before 1997](https://pubs.opengroup.org/onlinepubs/7908799/xsh/getpagesize.html) (see the copyright header).]

Reply:
not sure the mange page in linux says that with sysconf  you are more portable but getpagesize is still available in linux.